src/vote/domain/candidate.ts

```diff
@@ -1,10 +1,17 @@
+import { CandidateEntity, StarEntity } from '@prisma/client';
+
 export class Candidate {
-  constructor(
-    private readonly _id: number,
-    private _starId: number,
-    private _name: string,
-    private _voteCount: number,
-  ) {}
+  id: number;
+  starId: number;
+  name: string;
+  voteCount: number;
+
+  constructor(id: number, starId: number, name: string, voteCount: number) {
+    this.id = id;
+    this.starId = starId;
+    this.name = name;
+    this.voteCount = voteCount;
+  }
```

Comment:
swagger 혹은 validation 데코레이터가 붙을 예정이 아니라면,
TS 매개변수 프로퍼티 문법을 이용하여, 간결하게 작성해볼 수 있을 것 같습니다.

```suggestion
  constructor(
    public id: number,
    public starId: number,
    public name: string,
    public voteCount: number,
  ) {}
```

제 개인적인 추천입니다.
(nest 에서 컨트롤러 서비스 등에서 쓰이는 패턴이기도 하고요)

- https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties
- https://docs.nestjs.com/providers#dependency-injection